api-2.2/pom.xml

```diff
@@ -0,0 +1,151 @@
+<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+	xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/maven-v4_0_0.xsd">
+	<modelVersion>4.0.0</modelVersion>
+
+	<parent>
+		<groupId>org.openmrs.module</groupId>
+		<artifactId>reporting</artifactId>
+		<version>1.19.0-SNAPSHOT</version>
+	</parent> 
+
+	<artifactId>reporting-api-2.2</artifactId>
+	<packaging>jar</packaging>
+	<name>reporting Module api 2.2</name>
+	<description>2.2 api project for reporting</description>
+	
+	<properties>
+	   <jackson.version>2.5.2</jackson.version>
+		<openMRSVersion>2.2.0</openMRSVersion>
+	</properties>
+
+	<dependencies>
```

Comment:
Can you explain all of these dependencies?  I don't think this is what we want.  See the existing 1.9 and 2.0 submodules and what their poms look like.  I'd think we should mostly be able to copy those.

Reply:
oh sorry , i had innitially used a diffent way to build up this POM , but i have now cleared all the unnecesary dependencies